### PR TITLE
NEW : add coefficient on unit  ht   line  in expenseReport card

### PR DESCRIPTION
**Pull request by atm-jpb**
# NEW| add coef on input ht line expenseReport

In expense report card. 
when choosing the type of mileage expense report, automatically add the coefficient relating to the qty
it 'will provide  the right calculation on add action via object
to discuss (this version will not take offset on calculation )

### CODE REVIEW

**Comment by john-botella** [comment]
Set as draft instead of WIP

File: htdocs/expensereport/card.php [hunk 1]
```diff
@@ -2559,7 +2559,7 @@
 
 				// Quantity
 				print '<td class="right inputqty">';
-				print '<input type="text" min="0" class="right maxwidth50" name="qty" value="'.dol_escape_htmltag($qty ? $qty : 1).'">'; // We must be able to enter decimal qty
+				print '<input type="text" min="0" class="right maxwidth50" id="input_qty" name="qty" value="'.dol_escape_htmltag($qty ? $qty : 1).'">'; // We must be able to enter decimal qty
```
Comment (eldy) [inline htdocs/expensereport/card.php:2562]: A field with id="input_qty" has already be added line 2300.
an html id must be used only once per page.

Comment (atm-jpb) [reply htdocs/expensereport/card.php:2562]: maybe i'm wrong but this two ids declaration never appear at the same moment.
$action != 'editline'  in one case and $action == 'editline' for the other

Comment (atm-jpb) [reply htdocs/expensereport/card.php]: i made the changes on my last commit.

**Pull request merged**